server/model/monitor.js

```diff
@@ -271,6 +272,14 @@ class Monitor extends BeanModel {
         return Boolean(this.grpcEnableTls);
     }
 
+    /**
+     * Parse to boolean
+     * @returns {boolean}
+     */
+    getCacheBust() {
+        return Boolean(this.cacheBust);
+    }
+
```

Comment:
Is a new method really needed here? Given it is only used once I think, to me it seems cleaner just to put `Boolean(this.cacheBust)` where ever it is called.

```suggestion
```

Reply:
I did it this way to be consistent with all the other Boolean values in this file. 
`/**
     * Is the TLS expiry notification enabled?
     * @returns {boolean}
     */
    isEnabledExpiryNotification() {
        return Boolean(this.expiryNotification);
    }

    /**
     * Parse to boolean
     * @returns {boolean}
     */
    getIgnoreTls() {
        return Boolean(this.ignoreTls);
    }

    /**
     * Parse to boolean
     * @returns {boolean}
     */
    isUpsideDown() {
        return Boolean(this.upsideDown);
    }
`